.drone.star

```diff
@@ -3050,6 +2958,25 @@ def pipelineSanityChecks(ctx, pipelines):
     for image in images.keys():
         print(" %sx\t%s" % (images[image], image))
 
+def logAcceptanceTestsScreenshotsResult(suite):
+    return [{
+        "name": "log-acceptance-tests-screenshot",
+        "image": OC_UBUNTU,
+        "commands": [
+            "cd %s/tests/acceptance/reports/screenshots/" % dir["web"],
+            'echo "To see the screenshots, please visit the following path"',
+            'for f in *.png; do echo "### $f\n" \'!\'"(https://cache.owncloud.com/public/${DRONE_REPO}/${DRONE_BUILD_NUMBER}/screenshots/$f) \n"; done',
```

Comment:
`https://cache.owncloud.com/public` is supposed to be public anyway, so there should not be any problem exposing this by hardcoding it here.